### ISSUE TITLE
Main window does not appear on-screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#92]
```
1. Start autokey-gtk
2. Right click notification icon
3. Click on "show main window"

No window appears. 

Please use labels and text to provide additional information.

Traceback (most recent call last):
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/notifier.py", line 133, in on_show_configure
    self.app.show_configure()
  File "/usr/lib/python2.6/dist-packages/autokey/gtkapp.py", line 211, in show_configure
    self.configWindow = ConfigWindow(self)
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 594, in __init__
    self.__initStack()
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 1218, in __initStack
    self.blankPage = BlankPage(self)
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 193, in __init__
    builder = get_ui("blankpage.xml")
  File "/usr/lib/python2.6/dist-packages/autokey/gtkui/configwindow.py", line 48, in get_ui
    builder.add_from_file(uiFile)
glib.GError: Failed to open file 
'/usr/lib/python2.6/dist-packages/autokey/gtkui/data/blankpage.xml': No such 
file or directory


```

Original issue reported on code.google.com by `Luke.Faraone` on 22 Oct 2010 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
I think the issue is that `blankpage.xml` was not checked in along with r303, 
which references it. 
```

Original comment by `Luke.Faraone` on 22 Oct 2010 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
Bugger, there's always something. Bloody subversion...
```

Original comment by `cdekter` on 23 Oct 2010 at 12:02

**Comment by GoogleCodeExporter** [comment]
```
Issue 93 has been merged into this issue.
```

Original comment by `cdekter` on 23 Oct 2010 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in packaging versions 0.71.1-1
```

Original comment by `cdekter` on 23 Oct 2010 at 11:13
- Changed state: **Fixed**
